### PR TITLE
feat: unit input in mantine

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: #6264 
Closes: https://github.com/lightdash/lightdash/issues/6267
Unblocks: https://github.com/lightdash/lightdash/issues/6265

CleanShot 2023-07-25 at 21 09 51@2x

### CODE REVIEW

File: packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Grid/index.tsx [hunk 1]
```diff
@@ -1,25 +1,19 @@
...
     bottom: '30px', // pixels from bottom (makes room for x-axis)
-};
+} as const;
 
-enum Positions {
-    Left = 'left',
-    Right = 'right',
-    Top = 'top',
-    Bottom = 'bottom',
-}
```
Comment (magnew) [inline packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Grid/index.tsx:22]: I've never seen this const...as const construct. What does that do?

File: packages/frontend/src/components/common/UnitInput/index.tsx [hunk 2]
```diff
@@ -77,33 +77,25 @@ const UnitInput = forwardRef<HTMLInputElement, UnitInputProps>(
...
-                    inputRef.current = input;
-
-                    if (typeof ref === 'function') {
-                        ref(input);
-                    } else if (ref) {
-                        ref.current = input;
-                    }
-                }}
+            <TextInput
+                ref={mergeRefs(inputRef, ref)}
```
Comment (magnew) [inline packages/frontend/src/components/common/UnitInput/index.tsx:81]: Nice

**Review by magnew — APPROVED** [review]
Tested. Looks great.

File: packages/frontend/src/components/ReactHookForm/UnitInput.tsx
[the remark this answers is not in the document]
Comment (IrakliJani) [reply packages/frontend/src/components/ReactHookForm/UnitInput.tsx:1]: nope! kill it please

**Pull request merged**